session-name-1/activities/activity-folder-1/README.md

```diff
@@ -0,0 +1,17 @@
+# Name: Web/Binary: Session Name: Challenge Name
+
+## Description
+
+Get the flag from [url]http://...:.../[/url].
```

Comment:
```suggestion
Get the flag from [challenge-name](http://...:.../).
```
This is how we did it on CyberEDU

Reply:
But I don't think we should keep a description here if it doesn't bring anything useful - it only contains variables which might give us trouble updating

Reply:
Done.